calibrate.py

```diff
@@ -582,13 +586,14 @@ def calibrate(self):
     def run(self):
         if 'capture' in self.args.mode:
             try:
-                # if self.args.imageOp == 'delete':
-                shutil.rmtree('dataset/')
+                if self.args.imageOp == 'delete' and Path('dataset').exists():
```

Comment:
I was thinking of deleting the dataset the old by default when they start a new calibration procedure by default.

Reply:
This would work better in case if the users had messed up with files inside the folder. like adding additional images inside which will get used in calibration in the new execution unless deleted.

Reply:
Sure, should we remove the `imageOp` then?

Reply:
Yup.